website/www/site/content/en/get-started/try-beam-playground.md

```diff
@@ -25,7 +26,7 @@ You can try the available Apache Beam examples at
 
 ## Beam Playground WordCount Example
 
-{{< playground "true" "SDK_JAVA/PRECOMPILED_OBJECT_TYPE_EXAMPLE/MinimalWordCount" "" "700px">}}
+{{< playground "true" "SDK_JAVA/PRECOMPILED_OBJECT_TYPE_EXAMPLE/MinimalWordCount" "" "700px" "playgroundIframe">}}
```

Comment:
I have another vision for this. We often will need Playground with tabs for multiple languages. A shortcode should be like

```
{{< playground >}}
  {{< playground_snippet sdk="java" example="SDK_JAVA/PRECOMPILEDOBJECT_TYPE_EXAMPLE/MinimalWordCount" ... >}}
  {{< playground_snippet sdk="python" ... >}}
{{< /playground >}}
```

Reply:
The idea is that `playground_snippet` inserts a `div` with `data-xxx` attributes or something like this.
`playground` shortcode then inserts a JS that looks for these nested dives. If only one is given, it renders a single iframe. If multiple are given, it creates tabs.

If you need to push this fast, can you rename `playground` shortcode to `playground_iframe` or something like so we can reserve `playground` shortcode for the above solution?

Reply:
renamed playground shortcode to playground_iframe